tee/prototype/epoch.go

```diff
@@ -50,6 +51,15 @@ func cloneBalances(a map[common.Address]*Bal) map[common.Address]*Bal {
 	return b
 }
 
+func (e *Epoch) applyExits(exiters exitersSet) {
+	log := log.WithField("epoch", e.Number)
```

Comment:
```pre
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xb5cfba]

goroutine 74 [running]:
github.com/perun-network/erdstall/tee/prototype.(*Epoch).applyExits(0x0, 0x0, 0x0, 0x0)
        /home/vados/go/src/github.com/perun-network/erdstall/tee/prototype/epoch.go:55 +0x3a
github.com/perun-network/erdstall/tee/prototype.(*Enclave).txRoutine(0xc00014a360, 0xc0000178c0, 0xc0001e21e0, 0x0, 0x854b39, 0x0)
        /home/vados/go/src/github.com/perun-network/erdstall/tee/prototype/epochproc.go:98 +0xd17
github.com/perun-network/erdstall/tee/prototype.(*Enclave).epochProcessor.func2(0xc000017860, 0xc00014a360, 0xc0000178c0, 0xc0001e21e0, 0xc00025a100)
        /home/vados/go/src/github.com/perun-network/erdstall/tee/prototype/epochproc.go:48 +0x4c
created by github.com/perun-network/erdstall/tee/prototype.(*Enclave).epochProcessor
        /home/vados/go/src/github.com/perun-network/erdstall/tee/prototype/epochproc.go:47 +0x59e
```
Can the epoch be `nil`?! @ndzik

Reply:
Ok you fixed it.

Reply:
It can indeed, in the very first two phases.